content/quickstart/python/metrics.md

```diff
@@ -89,6 +78,8 @@ def main():
     while True:
         line = sys.stdin.readline()
         print(line.upper())
+
+main()
```

Comment:
Please make this
```python
if __name__ == '__main__':
      main()
```
as that's the more idiomatic way of writing Python code :)

Reply:
Thank you for reviewing!